src/storage/pubsub/listener.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "index.h"
+
+#include <stream/cc_sockio.h>
+
+/*
+ * a listener is a client that has subscribed to at least one channel
```

Comment:
The connection is still around, but no longer a listener. This is explained on redis.io in the [pubsub section](https://redis.io/topics/pubsub):

> unsubscribe: means that we successfully unsubscribed from the channel given as second element in the reply. The third argument represents the number of channels we are currently subscribed to. When the last argument is zero, we are no longer subscribed to any channel, and the client can issue any kind of Redis command as we are outside the Pub/Sub state.